3scale-operator/tasks/main.yml

```diff
@@ -147,8 +138,12 @@
     state: absent
     api_version: v1
     kind: Secret
-    name: aws-auth
+    name: 3scaleportal
     namespace: "{{ target_namespace }}"
     wait: true
     wait_timeout: 60
     wait_sleep: 5
+- name: Create s3 bucket for 3Scale API Management
+  amazon.aws.s3_bucket:
+    name: 3scalebucket
+    state: absent
```

Comment:
we need a task to also uninstall the apicast operator

Reply:
Done.